### ISSUE TITLE
networking.k8s.io/v1beta1 Ingress is deprecated in v1.19+

### ISSUE THREAD

**Issue by jahio** [tweedproject/tweed#20]
When applying `eval.yml` to a Kubernetes cluster (on localhost, created by [kind](https://kind.sigs.k8s.io/)) version 1.20.2, I'm seeing the following deprecation warning:

```
Warning: networking.k8s.io/v1beta1 Ingress is deprecated in v1.19+, unavailable in v1.22+; use networking.k8s.io/v1 Ingress
```

Probably a simple fix; this issue is just to document that it needs to be done for future compatibility.

**Comment by vinaykudari** [comment]
This can be closed. This was fixed in [be8df21](https://github.com/tweedproject/tweed/commit/be8df210f5a2b22abef56b0258aca910c6e0b246)

**Comment by jahio** [comment]
Thanks Vinay!
